lib/src/widgets/chatview_state_widget.dart

```diff
@@ -31,7 +35,8 @@ class ChatViewStateWidget extends StatelessWidget {
             children: [
               Text(
                 (chatViewStateWidgetConfig?.title
-                    .getChatViewStateTitle(chatViewState))!,
+                        .getChatViewStateTitle(chatViewState)) ??
+                    fallbackTitle,
```

Comment:
This will compulsorily show the title even when the end user may not want it. Please revert this change.

Reply:
I have a good argument against the current usage:
```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following _TypeError was thrown building ChatViewStateWidget(dirty):                                                                                       
Null check operator used on a null value                                                                                                                       
                                                                                                                                                               
The relevant error-causing widget was:                                                                                                                         
  ChatViewStateWidget                                                                                                                                          
  ChatViewStateWidget:file:///home/jbadstuebner/Dokumente/git/github.com/jonasbadstuebner/flutter_chatview/remove-arbitrary-padding/lib/src/widgets/chat_view.dart:215:46
                                                                                                                                                               
When the exception was thrown, this was the stack:                                                                                                             
#0      ChatViewStateWidget.build (package:chatview/src/widgets/chatview_state_widget.dart:38:59)
#1      StatelessElement.build (package:flutter/src/widgets/framework.dart:5557:49)         
#2      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5487:15)                                                                   
#3      Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)                                                                                    
#4      StatelessElement.update (package:flutter/src/widgets/framework.dart:5563:5)
#5      Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)            
#6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512:16)                                                                   
#7      Element.rebuild (package:flutter/src/widgets/framework.dart:5203:7)                                                                                    
#8      ProxyElement.update (package:flutter/src/widgets/framework.dart:5816:5)    
#9      Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)            
#10     Element.updateChildren (package:flutter/src/widgets/framework.dart:3976:32)         
#11     MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6929:17)                                                              
#12     Element.updateChild (package:flutter/src/widgets/framework.dart:3827:15)   
#13     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776:14)
[…]
```

So the `Text` widget is always rendered, even if the value is null. So setting it to null will crash your app currently, instead of not showing an "error" or "loading" state.

I suggest we fall back to a default title and just hide the Text widget if the title is `""`.